chainer/links/normalization/layer_normalization.py

```diff
@@ -0,0 +1,53 @@
+from chainer import initializers
+from chainer import link
+
+from chainer.functions.array import broadcast
+from chainer.functions.math import sqrt
+from chainer.functions.math import square
+from chainer.functions.math import sum
+from chainer.links.connection import scale
+
+
+class LayerNormalization(link.Chain):
+
+    """Layer normalization layer on outputs of linear functions.
+
+    This is a link of "Layer Normalization". This layer
+    normalizes input units by statistics along the second axis,
+    scales and shifts them with :class:`~chainer.links.Scale`.
+
+    Args:
+        size (int): Size of input units.
+
+    Attributes:
+        scale (~chainer.links.Scale): Layer to scale and shift
+            units after normalization.
+        eps (float): Epsilon value for numerical stability.
+
+    See: `Layer Normalization <https://arxiv.org/abs/1607.06450>`_
+    """
+
+    def __init__(self, size, eps=1e-6, initial_gamma=None, initial_beta=None):
+        super(LayerNormalization, self).__init__(
+            scale=scale.Scale(axis=1, W_shape=(size, ), bias_term=True),
+        )
+        if initial_gamma is None:
+            initial_gamma = initializers.One()
+        initializers.init_weight(self.scale.W.data, initial_gamma)
+        if initial_beta is None:
+            initial_beta = initializers.Zero()
+        initializers.init_weight(self.scale.bias.b.data, initial_beta)
+        self.eps = eps
+
+    def normalize(self, x):
```

Comment:
Is this intended to be a private method? If so, consider renaming to something like `_normalize()` to give users a hint that it is not part of documented API.